src/msg.rs

```diff
@@ -1,3 +1,9 @@
+/*
+    These constants come from:
+
+    https://github.com/openssh/openssh-portable/blob/master/authfd.h
```

Comment:
We could point to the IETF draft instead, since it gives a neater description (and because we can reference it in other places of the agent too, for referencing).

```suggestion
    https://datatracker.ietf.org/doc/draft-ietf-sshm-ssh-agent/
m```

Reply:
Great idea. Updated & added the relevant section for anyone quickly searching the doc, let me know what you think.

Reply:
Nice work!